src/reachy2_sdk/media/camera.py

```diff
@@ -80,7 +125,17 @@ def get_extrinsics(self, view: CameraView = CameraView.LEFT) -> Optional[npt.NDA
     def pixel_to_world(
         self, u: int, v: int, z_c: float = 1.0, view: CameraView = CameraView.LEFT
     ) -> Optional[npt.NDArray[np.float64]]:
-        """Compute the XYZ position given a uv pixel in a camera view. If known the depth z_c (meter) may be provided"""
+        """Convert pixel coordinates to XYZ coordinate in Reachy coordinate system.
+
+        Args:
+            u: The x-coordinate (pixel) in the camera view.
```

Comment:
maybe we could add that u is the horizontal axis and v the vertical one, to be even clearer ?